src/loader-code.coffee

```diff
@@ -284,6 +623,35 @@ Let us know and we will get back to you once we fix the issue.
   error = (s) ->
     window.console.error "widgetLoader: " + s  if window["console"] isnt `undefined`
 
+  addReporterrorListeners = () -> 
+    onSubmitComplete = (error) ->
+      contactResponse = document.getElementById('contact-response')
+      contactForm.clazz('submited');
+      contactBtn.disabled = false;
+      if error
+        contactResponse.innerHTML = '<div class="state-error">Sorry. Could not submit the error report.</div>'
+      else
+        contactResponse.innerHTML = '<div class="state-success">Thanks for submitting your error report!</div>'
+      return
+
+    contactForm = $s('#contact-form')
+    contactBtn = $s('#btn-submit')
+    contactBtn.on 'click',(e)=>
+      e.preventDefault()
+      myFirebaseRef = new Firebase('https://epiclogger.firebaseio.com/errors')
+      id = window.ELopts.widget_url
+      currentdate = new Date
+      datetime = currentdate.getDate() + '/' + currentdate.getMonth() + 1 + '/' + currentdate.getFullYear() + ' @ ' + currentdate.getHours() + ':' + currentdate.getMinutes() + ':' + currentdate.getSeconds()
+      myFirebaseRef.push {
+        'id': id
+        'email': contactForm.InputEmail.value
+        'notes': contactForm.InputMessage.value
+        'timestamp': datetime
+      }, onSubmitComplete
```

Comment:
this function needs to be refactored so we use our data instead of this code here to send the information to. Considering we don't have jquery we need to look at some jquery similar plugins that are small and make the ajax call or try to use simple javascript to make the request. Let me know if you have any suggestions on this.